config/Config-kernel.in

```diff
@@ -1125,6 +1125,12 @@ if KERNEL_IP_MROUTE
 
 endif
 
+config KERNEL_MPTCP
+	bool "Enable MultiPath TCP support"
+	default y
```

Comment:
MPTCP currently isn't enabled in the kernel, so the default should be n.

Also please add info (in the commit) about how much the size of the kernel increases with this enabled.